src/tools/sdk/dart_sdk_archive/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:pedantic/analysis_options.yaml
```

Comment:
include: package:pedantic/analysis_options.1.7.0.yaml to not get travis build failures when a new version of pedantic is released